src/App.tsx

```diff
@@ -3,43 +3,48 @@ import React, {useEffect, useState} from 'react';
 import './App.less';
 import {fetchData} from "./utils/fetch";
 import {Bruker, DialogData} from "./utils/typer";
-import {HenvendelseList} from "./view/HenvendelseList";
-import { UserInfoContext} from "./Context";
-import {DialogBanner} from "./view/DialogBanner";
-
 
 import './App.less';
 import {DialogOverview} from "./view/DialogOverview";
-
+import {Dialog} from "./view/Dialog";
+import {DialogBanner} from "./view/DialogBanner";
+import { UserInfoContext} from "./Context";
 
 const App = () => {
 
     const [dialogListe, setDialogListe] = useState<DialogData[] | undefined>(undefined);
-
     const [userInfo, setUserInfo] = useState<Bruker | undefined>(undefined);
 
     useEffect(() => {
         fetchData<DialogData[]>("/veilarbdialog/api/dialog", {method: 'get'})
             .then(res => setDialogListe(res));
         fetchData<Bruker>("/veilarboppfolging/api/oppfolging/me", {method: 'get'})
             .then(res => setUserInfo(res))
+
     }, []);
 
 
-    return (<>
+    return (
+        <>
             <div className="app">
                 <DialogBanner/>
                 <UserInfoContext.Provider value={userInfo}>
-                {dialogListe === undefined ? null : <DialogOverview dialogData={dialogListe}/>}
-                <div className="henvendelseList"> {dialogListe === undefined ? null :
-                    <HenvendelseList henvendelseDataList={dialogListe[0].henvendelser}/>}</div>
+                    <div className="App-body">
+                        <div className="dialog-list">
+                            { dialogListe === undefined? null : <DialogOverview dialogData={dialogListe}/> }
```

Comment:
Jeg bruker vanligvis å flytte ansvaret for null-sjekkingen inn i komponentene for å unngå tenary-if i koden.
Da kan koden her forenkles til `<DialogOverview dialogData={dialogListe} />`. Inne i selve komponenten gjør man bare en rask sjekk i starten, e.g
```
function DialogOverview(props: Props) {
  if (!props.dialogData) {
    return null;
  }
  // Resten av logikken her
}
```